src/components/playground/CustomizerSlider.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Radium from 'radium';
+
+@connect(state => state.playground)
+@Radium
+class CustomizerSlider extends React.Component {
```

Comment:
Not so far, some features were going to have a slider control but it's safe to remove, it's empty anyway.